databases/week2/JSONExercise.sql

```diff
@@ -0,0 +1,50 @@
+CREATE TABLE author(
+  id INT PRIMARY KEY NOT NULL AUTO_INCREMENT,
+  name VARCHAR(20) NOT NULL
+)
+
+INSERT INTO author(id, name) VALUES(1, "James Smith"),
+(2, "Jane Jones"),
+(3, "Aliya Awad"),
+(4, "Igor Vladimir"),
+(5, "Kim Jensen")
+
+CREATE Table article(
+  id INT PRIMARY KEY NOT NULL AUTO_INCREMENT,
+  title VARCHAR(255) NOT NULL,
+  content VARCHAR(255)
+);
+
+
+INSERT INTO article(id, title, content) 
+VALUES(1, "BREAKING NEWS: Water is wet!", "Scientists have discovered that water is wet, it's amazing what.... ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur."),
+(2, "Heavy Snowfall Expected this Weekend", "Lots of snow is expected... Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum."),
+(3, "BREAKING NEWS: These 10 Clickbait Titles Are Bad for Your Health, Number 7 Will SHOCK You!", "Haha, you clicked! Minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat")
+
+CREATE Table tags(
+  id INT PRIMARY KEY AUTO_INCREMENT,
+  name VARCHAR(20),
+  article_id INT NOT NULL,
```

Comment:
A tag can be in several articles. This means that you would have repeated names in this table, one for each article it appears. This means it is a many to many relation. An article can have many tags, and a tag can appear in many articles.

Reply:
Thanks to point it out. I made the changes in ERD and Data. Also created a junction table between tags and article to establish many-to-many relationship.